src/states/node.rs

```diff
@@ -871,8 +871,8 @@ impl Node {
                                                      src_name,
                                                      dst)
             }
-            (CandidateApproval { candidate_id, client_auth, sections }, Section(_), Section(_)) => {
-                self.handle_candidate_approval(candidate_id, client_auth, sections, outbox)
+            (CandidateApproval { candidate_id, client_auth, .. }, Section(_), Section(_)) => {
+                self.handle_candidate_approval(candidate_id, client_auth, outbox)
```

Comment:
I think the plan was to leave that in for safety: If the section can't even accumulate the `CandidateApproval` including the routing table, it's better to not even add the candidate to anyone's routing table.
The problem was that the routing tables could change between sending and handling that message, and then it would be better to put the new routing table into the `NodeApproval`. If _that_ doesn't accumulate, the nodes do put the candidate in their routing tables and have to rely on the candidate disconnecting.